job/server/src/main/java/alluxio/master/AlluxioJobMasterProcess.java

```diff
@@ -203,6 +206,10 @@ protected void startServingRPCServer() {
           ServerUserState.global());
       registerServices(serverBuilder, mJobMaster.getServices());
 
+      // Add journal master service.
+      serverBuilder.addService(alluxio.grpc.ServiceType.JOURNAL_MASTER_CLIENT_SERVICE,
```

Comment:
Will it better to put in `JobMaster.getServices`?

Reply:
Theoretically, It could have been put to any master. However it's very confusing as non of them owns the journalsystem. Only master processes owns them so that's why I'm doing the insertion at that level for both alluxio and job_master processes.

Reply:
Make sense